eng/GenerateAndExecuteMockTest.ps1

```diff
@@ -429,8 +418,5 @@ function  MockTestInit {
 }
 
 # Generate & Run All SDK
-$commitId = "322d0edbc46e10b04a56f3279cecaa8fe4d3b69b"
 $GenerateNewSDKs = $true
-$NpmInit = $true
```

Comment:
Hi @dvbb ，

Some strange error happen in pipeline since June. Would you help add memory setting here:
`$env:NODE_OPTIONS="--max_old_space_size=8192"`

And also add `--max-memory-size=8192` as autorest option at LINE 150
https://github.com/Azure/autorest.csharp/pull/2313/files#diff-99afbb031007d216ce7e29e36b91da3090f88709ea4703e92bd5429d8669420cL156